dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/AggregatedDataRequest.kt

```diff
@@ -16,5 +16,6 @@ data class AggregatedDataRequest(
     val reportingPeriod: String?,
     val dataRequestCompanyIdentifierType: DataRequestCompanyIdentifierType,
     val dataRequestCompanyIdentifierValue: String,
+    val requestStatus: RequestStatus,
```

Comment:
Parameter requestStatus is not part of the data class's documentation - necessary?

Reply:
Not necessary but better to have it. Done.